### DISCUSSION TITLE
使用v2ray导致onedrive企业版无法登录

### DISCUSSION THREAD

**Discussion by Firestar-Reimu** [v2fly/v2ray-core#2014] [Q&A]
![image](https://user-images.githubusercontent.com/65082432/192524682-ade22645-026d-4cb5-bd78-e58d9d0eb182.png)

我有一个onedrive个人版（买电脑自带的）和一个onedrive企业版（microsoft开发人员），如果我开启v2ray，采取全局代理，会导致onedrive企业版无法登录

关掉v2ray并重启onedrive企业版就可以自动登录了

v2ray 版本：5.1.0和4.x版本

**Comment by jiangjiangflies** [comment]
我也遇到同样问题了，楼主解决了吗？

**Comment by Firestar-Reimu** [comment]
现在好像又没事了

![image](https://user-images.githubusercontent.com/65082432/202609474-fa9f0904-e149-428c-891f-ed404f83e67d.png)

![image](https://user-images.githubusercontent.com/65082432/202609573-2781e53a-9179-4725-8076-d0d72b8b1944.png)

我不知道是不是和使用的DNS有关

**Comment by jiangjiangflies** [comment]
我的还是不行。只要开了v2rayN系统代理（绕过大陆IP路由规则），onedrive（世纪互联版）就没法登录。清除系统代理就好了。

**Reply by gachoyi** [reply]
@jiangjiangflies 找到解决方法了吗？这个应该是v2rayN的问题。

**Reply by jiangjiangflies** [reply]
我这边解决了。在软件设置里，找到系统代理设置，将login.partner.microsoftonline.cn加入例外。

**Reply by gachoyi** [reply]
Nice,可以了，我一直把这个地址放在路由里，没有用。

**Reply by TongYoungG** [reply]
有遇到个人版无法登录的情况吗

**Reply by gydx6** [reply]
> 我这边解决了。在软件设置里，找到系统代理设置，将login.partner.microsoftonline.cn加入例外。

谢谢你的方法，我是将“login.partner.microsoftonline.cn;login.microsoftonline.com”这两个网址加到“系统代理设置”后解决

**Reply by seflerZ** [reply]
添加了还是不行

**Comment by yangzexia** [comment]
https://www.cnblogs.com/liuhanxu/p/17620600.html

**Comment by Firestar-Reimu** [comment]
> 在代理客户端打开【设置】--> 【解除Win10 UWP应用回环代理限制】--> 勾选上【工作或学校帐户】，然后保存即可成功登录OneDrive。

那哪个代理客户端有这种设置呢？

**Reply by yangzexia** [reply]
windows上v2rayN，clash for windows都可以

**Comment by Firestar-Reimu** [comment]
好吧，我在windows/linux都用的v2rayA
